data/state/accountsDB.go

```diff
@@ -14,7 +16,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go/marshal"
 )
 
-const batchWriteDelay = time.Second * 2
+var numCheckpointsKey = []byte("state checkpoint")
```

Comment:
you can move it to core/constants.go and make it exported and use it everywhere because there are more places / packages where you use this

Reply:
Actually, this is the only place where it is/will be used, because here is where you save it to db/recover it from db.